### ISSUE TITLE
[dev.icinga.com #10916] Unable to create user group "Administrators"

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#2227]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/10916

**Created by gilbertaispuro on 2015-12-28 17:58:23 +00:00**

Assignee: _(none)_
Status: _Closed_  (closed on _2016-02-18 15:59:15 +00:00_)
Target Version: _(none)_
Last Update: _2016-02-18 15:59:15 +00:00 (in Redmine)_

---

Hello,

I am having issues setting up Icingaweb.

I can go through the whole setup with no issues but when I click finish at the last page, i receive the following prompt:

User Group Backend configuration has been successfully written to: /etc/icingaweb2/groups.ini
Unable to create user group "Administrators". An error occured:
ERROR: Zend\_Db\_Statement\_Exception in /usr/share/php/Zend/Db/Statement/Pdo.php:235 with message: SQLSTATE\[42S02\]: Base table or view not found: 1146 Table 'icingaweb2.icingaweb\_group' doesn't exist, query was: SELECT COUNT AS cnt FROM icingaweb\_group AS g WHERE (g.name = 'Administrators')

I have no SQL experience by the way.

Please help.

Thank you!

**Attachments**

* [IcingaWeb_Issue.JPG](https://dev.icinga.com/attachments/download/2974/IcingaWeb_Issue.JPG) gilbertaispuro - _2015-12-28 17:57:55 +00:00_

---

**Relations**:

* relates [#10916](https://dev.icinga.com/issues/10916)

**Comment by meilaniagnesia** [comment]
Hello, I am facing this same issue

User Group Backend configuration has been successfully written to: /etc/icingaweb2/groups.ini
Unable to create user group "Administrators". An error occured:
ERROR: Zend_Db_Adapter_Exception in /usr/share/icingaweb2/library/vendor/Zend/Db/Adapter/Pdo/Abstract.php:139 with message: SQLSTATE[28000] [1045] Access denied for user 'apache'@'localhost' (using password: YES)

Anyone could help fix this issue will be highly appreciated. Thank you.

**Comment by corsairetc** [comment]
Hello, I have same problem.
`Unable to create user group "Administrators". An error occured:
ERROR: Zend_Db_Statement_Exception in /usr/share/icingaweb2/library/vendor/Zend/Db/Statement/Pdo.php:225 with message: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'icinga_users.icingaweb_group' doesn't exist, query was: SELECT COUNT(*) AS cnt FROM icingaweb_group AS g WHERE (g.name = 'Administrators')
`
Mysql schema:
`MariaDB [(none)]> show databases;
+--------------------+
| Database           |
+--------------------+
| icinga_users       |
| icingadb           |
| information_schema |
| mysql              |
| performance_schema |
+--------------------+
`
I have very low skills with sql. Thank you for any advice.
I try it install on fresh Debian 9 x64 installation in VMs.

**Comment by pkhadka56** [comment]
Well, I changed database name other than icingaweb2 and icinga2 and it worked. Just change database name.

**Comment by jlownie** [comment]
I had the same issue, I was getting the same error as in gilbertaispuro's initial post and then I did the same as pkhadka56 - I created a new user, gave them access to the icingaweb2 DB then clicked back through the config wizard and changed the DB user for icingaweb2 to the new user, and then the wizard completed successfully.

I checked that the normal user was valid and could login using the mysql cli.  It looks like a bug to me.  I'm using Ubuntu 18.04, I installed Icinga from the icinga repos.

ii  icinga2                                               2.11.3-1.bionic
ii  icinga2-bin                                           2.11.3-1.bionic
ii  icinga2-common                                        2.11.3-1.bionic
ii  icinga2-doc                                           2.11.3-1.bionic
ii  icinga2-ido-mysql                                     2.11.3-1.bionic
ii  icingacli                                             2.7.3-1.bionic
ii  icingaweb2                                            2.7.3-1.bionic
ii  icingaweb2-common                                     2.7.3-1.bionic
ii  icingaweb2-module-doc                                 2.7.3-1.bionic
ii  icingaweb2-module-monitoring                          2.7.3-1.bionic
